deploy/kube-config/riemann/riemann-service.json

```diff
@@ -0,0 +1,36 @@
+{
+  "apiVersion": "v1beta3",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "kubernetes.io/cluster-service": "true",
+      "name": "riemann-heapster"
+    },
+    "name": "riemann-heapster"
+  },
+  "spec": {
```

Comment:
We can route to InfluxDB via Riemann, and you probably typically would. It's our intent to do so. Setting up that configuration is a larger task than I wanted to take on in this PR.